.npm/new.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+const packageJson = require("./package");
+const fs = require("fs");
+const { download } = require("./download");
+const { execute } = require("./execute");
+const path = require("path");
+
+(async () => {
+  let args = process.argv;
+  args.shift(); // node
+  const dirPath = path.dirname(args[0]);
+  args[0] = "new"; // Replace ".../cca.js" with "new"
+
+  const ccaVersion = /^\d\.\d\.\d/.exec(packageJson.version)[0];
+  const binPath = `${dirPath}/create-comit-app_${ccaVersion}/create-comit-app`;
```

Comment:
This is still not going to work on windows (.exe is missing). Would be cool to fix this as part of this PR as-well!

Reply:
Can we please agree on the windows strategy first?